intyg-proxy-service/integration-pu/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/PuIntegrationService.java

```diff
@@ -1,12 +1,14 @@
 package se.inera.intyg.intygproxyservice.integration.pu;
 
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Service;
 import se.inera.intyg.intygproxyservice.integration.api.pu.Person;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuRequest;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuResponse;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuService;
 
 @Service
+@Profile("!fakepu")
```

Comment:
Consider creating a constant for fakepu to make it easier to track its usage.

Reply:
Thanks for the good suggestion! I will introduce this in the next PR!